server/src/game/Game.cpp

```
@@ -127,6 +131,15 @@ void game::Game::initECS() {
         _ecsManager->getComponentType<ecs::ColliderComponent>());
     _ecsManager->setSystemSignature<ecs::CollisionSystem>(collisionSignature);
 
+    Signature serverInputSignature;
```

Clang (14.0.6)

[warning] variable 'serverInputSignature' is not initialized

(cppcoreguidelines-init-variables)